src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -24,6 +24,7 @@
   --pf-c-dropdown__toggle--m-plain--BorderColor: transparent;
   --pf-c-dropdown__toggle--m-plain--Color: var(--pf-global--Color--200);
   --pf-c-dropdown__toggle--m-plain--hover--Color: var(--pf-global--Color--100);
+  --pf-c-dropdown__toggle-text--Width: calc(100% - var(--pf-global--spacer--lg));
```

Comment:
This addition broke the Dropdown - Icon Only case at http://patternfly-react.surge.sh/patternfly-4/components/dropdown.  :(

cc: @spadgett, @janwright73, @priley86

Reply:
Before: 
![screen shot 2019-03-05 at 2 41 25 pm](https://user-images.githubusercontent.com/895728/53832676-c962aa00-3f54-11e9-8ab3-a7fc0f9c538c.png)

After:
![screen shot 2019-03-05 at 2 41 05 pm](https://user-images.githubusercontent.com/895728/53832685-cd8ec780-3f54-11e9-80dd-ffdcd9d46344.png)

Reply:
Digging deeper, this kooks like this is bug in @patternfly/react-core.